bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -193,16 +218,18 @@ async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason:
             else:
                 raise
 
-        field = "reason" if nomination["active"] else "end_reason"
+        if nomination["active"]:
+            await ctx.send(":x: Cannot edit end reason of active nomination.")
+            return
 
-        self.log.trace(f"Changing {field} for nomination with id {nomination_id} to {reason}")
+        self.log.trace(f"Changing end reason for nomination with id {nomination_id} to {reason}")
```

Comment:
A repr would be nice here as well
```suggestion
        self.log.trace(f"Changing end reason for nomination with id {nomination_id} to {repr(reason)}")
```